models/comments.models.js

```diff
@@ -0,0 +1,11 @@
+const db = require('../db/connection')
+
+exports.fetchCommentsByArticleId = (id) => {
+    return db
+    .query(`SELECT *
+    FROM comments
+    WHERE comments.article_id=${id};`)
+    .then(({rows})=> {
```

Comment:
Should be using node-postgres parameterized query here rather than templating the id in directly